packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/quanticTimeframeFacet.html

```diff
@@ -1,90 +1,95 @@
 <template>
-  <slot>Timeframes</slot>
+  <slot><!-- Timeframes --></slot>
```

Comment:
why it is commented?

Reply:
Since it is an unnamed slot, I left a comment to indicate what the slot should contain.